capDL-tool/CapDL/MakeModel.hs

```diff
@@ -435,6 +440,8 @@ validObjPars (Obj MSIIrqSlot_T ps []) =
   subsetConstrs ps (replicate (numConstrs (Addr undefined)) (MSIIRQExtraParam undefined))
 validObjPars (Obj ARMIrqSlot_T ps []) =
   subsetConstrs ps (replicate (numConstrs (Addr undefined)) (ARMIRQExtraParam undefined))
+validObjPars (Obj ARMSID_T ps []) =
+  subsetConstrs ps (replicate (numConstrs (Addr undefined)) (SIDExtraParam undefined))
```

Comment:
Hmm, it's pretty harmless but it looks like quite a few of the other cases in this function were incorrectly copied. I'll fix the existing ones, but this should actually be
```suggestion
validObjPars (Obj ARMSID_T ps []) =
  subsetConstrs ps (replicate (numConstrs (SIDNumber undefined)) (SIDExtraParam undefined))
```